### PR TITLE
Update change-target-framework.md

### PR DESCRIPTION
**Pull request by jmacato**
I wonder if this is correct @grokys

### CODE REVIEW

File: src/AvaloniaUI.Net/wwwroot/docs/quickstart/change-target-framework.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ Order: 10
 ---
 By default, our project templates target the following frameworks:
 
-- **Visual Studio Extension**: Dual targets .NET 4.6.1 and .NET Core 2.0
-- **.NET Core Templates**: .NET Core 2.0
+- **Visual Studio Extension**: .NET Core 3.0
```
Comment (FoggyFinder) [inline src/AvaloniaUI.Net/wwwroot/docs/quickstart/change-target-framework.md:7]: `3.1`?

Comment (jmacato) [reply src/AvaloniaUI.Net/wwwroot/docs/quickstart/change-target-framework.md:7]: I saw the template as 3.0, but can you double check @FoggyFinder ?

Comment (FoggyFinder) [reply src/AvaloniaUI.Net/wwwroot/docs/quickstart/change-target-framework.md:7]: 3.1

https://github.com/AvaloniaUI/AvaloniaVS/blob/master/templates/AvaloniaMvvmApplicationTemplate/ProjectTemplate.csproj#L4

Comment (jmacato) [reply src/AvaloniaUI.Net/wwwroot/docs/quickstart/change-target-framework.md]: @FoggyFinder thanks alot :)

**Review by grokys — APPROVED** [review]
Looks to be correct. We should probably bump the .NET core version on the .net CLI templates.

**Pull request merged**